### PR TITLE
Call sync function in new thread

### PR DESCRIPTION
**Pull request by ScorpioCPH**
@kdada, PTAL.

### CODE REVIEW

File: src/ui/api/project.go [hunk 1]
```diff
@@ -114,9 +114,7 @@ func (p *ProjectAPI) Post() {
 	}
 
 	// sync registry for new project
-	if err := SyncRegistry(); err != nil {
-		log.Errorf("SyncRegistry error: %v", err)
-	}
+	go SyncRegistry()
```
Comment (kdada) [inline src/ui/api/project.go]:
```go
go func() {
	if err := SyncRegistry(); err != nil { 
		log.Errorf("SyncRegistry error: %v", err) 
	} 
}()
```

may be better

Comment (ScorpioCPH) [reply src/ui/api/project.go]: Good point.

**Comment by ScorpioCPH** [comment]
@kdada

**Comment by kdada** [comment]
LGTM

**Pull request merged**